### ISSUE TITLE
Ore blocks are always visible (x-ray like)

### ISSUE THREAD

**Issue by Bob74** [Snakefangox/modfest2022#16]
- Minecraft version: 1.16.1
- Fabric API version: 0.18.0+build.387-1.16.1
- Mod version: 1.9.6
- Patchouli version: 1.16-36-FABRIC
- And I'm using Phosphor, Lithium and Sodium

It seems that ores blocks are always rendered even when masked behind other blocks.

Here I am spectating under the ground (currently inside a stone block):
![Ores](https://i.imgur.com/NyjwFKN.png)

I don't know if it is bad for the performances but I don't think this is normal ^^

**Comment by snakefangox** [comment]
Yeah, looks like I screwed up the rendering. I'll have a proper look through and see what I can do to fix it. Thank you for reporting the issue.
